portfolio/src/main/webapp/script.js

```diff
@@ -250,3 +251,40 @@ function deleteComment(comment) {
   params.append('id', comment.id);
   fetch('/delete-comments', {method: 'POST', body: params});
 }
+
+/**
+ * Checks if the user is logged in and displays corresponding contents. 
+ */
+function checkLoginStatus() {
```

Comment:
If you're also displaying messages after checking the status, you might want to rename this.

`checkLoginStatus` would mean that all you do is check a true/false of whether the user is logged in. Nothing further.